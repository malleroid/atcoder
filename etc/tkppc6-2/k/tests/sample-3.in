4 5
31415 92653 58979 32384
