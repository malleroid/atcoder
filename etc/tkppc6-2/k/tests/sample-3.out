1
287242452
806145507
819893815
181831825
